README.md

```diff
@@ -61,3 +61,30 @@ should be rendered in a test.
 
 Compilation via 2 stage [Dockerfile](./Dockerfile).
 Image is build and pushed to hub.docker.com via [github action](.github/workflows/release.yml).
+
+### Simple setup guide for beginner devs
+
+Install git, docker, docker-compose and npm.
+
+To install all dependencies run
+```sh
+npm install
+```
+
+For the setup of docker run
+
+```sh
+sudo gpasswd -a $USER docker  
+newgrp docker
```

Comment:
Should not be necessary, the `docker` group is automatically created when docker is installed on all (almost all?) major distributions